docs/reporting/transactions.md

```diff
@@ -0,0 +1,46 @@
+# Retrieve transactions
+
+## Retrieve a list of transactions
+
+Returns a report of all sales transactions for a given merchant.
```

Comment:
No need for "for a given merchant". Also, "sales transactions" sound a bit weird, I'd leave just transactions as endpoint also returns refunds, fees, etc.

Reply:
fixed